backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/purchaseOrder/purchaseOrderComplexPaymentTerm.feature

```diff
@@ -42,7 +47,16 @@ Feature: Purchase order with complex payment term
     And metasfresh contains C_PaymentTerm_Break
       | Identifier | C_PaymentTerm_ID | Percent | OffsetDays | ReferenceDateType | SeqNo |
       | PTB1       | pt_PO            | 25      | 1          | OD                | 10    |
-      | PTB2       | pt_PO            | 75      | 0          | BL                | 20    |
+      | PTB2       | pt_PO            | 75      | 0          | LC                | 20    |
+    And metasfresh contains C_PaymentTerm
+      | Identifier | IsComplex |
+      | pt_PO_2    | Y         |
+    And metasfresh contains C_PaymentTerm_Break
```

Comment:
leave it like this for now, but next time 
* think to the reader of the tests
* think to the next people that will write tests,

so
* the step name shall be something like `metasfresh contains C_PaymentTerm_Break for pt_PO_2`
* and in columns:
   * C_PaymentTerm_ID shall not exists
   * SeqNo shall not be needed, you shall create the breaks in order they are listed here